base/array.jl

```diff
@@ -2,6 +2,38 @@
 
 ## array.jl: Dense arrays
 
+"""
+    Array{T}(dims)
+    Vector{T}(m)
+    Matrix{T}(m, n)
+
+`Array{T}(dims)` constructs an uninitialized dense array with element type `T`. `dims` may
+be a tuple or a series of integer arguments, each giving the size of the corresponding
+dimension. `Vector{T}(m)` and `Matrix{T}(m, n)` are stricly equivalent aliases to create one-
+and two-dimensional arrays.
+
+The syntax `Array(T, dims)` is also available, but deprecated.
+
+
```

Comment:
Splitting this docstring in two, both associated to `Array`, gives an error at build time:
```
docs/Docs.jl
error during bootstrap:
LoadError(at "sysimg.jl" line 342: UndefVarError(var=:STDERR))
```

Yet, doing the same at the REPL works fine.

@MichaelHatherly Is that expected?